Task 1/hacker-rank/10-days-of-javascript/24-buttons-container/Question.md

```diff
@@ -0,0 +1,8 @@
+##Objective
```

Comment:
Mention the link to the question and please follow the file structure mentioned.
--Task 1
----24-buttons-container
------Question.md
------Solution files